doc/user/content/sql/values.md

```diff
@@ -6,53 +6,68 @@ menu:
     parent: commands
 ---
 
-The `VALUES` expression constructs a relation from value expressions.
+`VALUES` constructs a relation from a list of parenthesized values.
```

Comment:
https://preview.materialize.com/materialize/30398/sql/values/

Reply:
It sounded too much like we were defining a word using the word -- "Values expression constructs ... from values expressions."   I don't mind making the last part "values expressions" but the previous sentence had a bit of "a thing is a thing" flavor to it.